src/main/java/org/araymond/joal/core/torrent/torrent/MockedTorrent.java

```diff
@@ -46,19 +48,20 @@ private MockedTorrent(final byte[] torrent, final boolean seeder) throws IOExcep
         } catch (final InvalidBEncodingException ex) {
             throw new IllegalArgumentException("Error reading torrent meta-info fields!", ex);
         }
-        this.infoHash = new InfoHash(this.getInfoHash());
+
+        this.torrentInfoHash = new InfoHash(this.getInfoHash());
     }
 
-    public InfoHash getTorrentInfoHash() {
-        return this.infoHash;
+    private MockedTorrent(byte[] data) throws IOException, NoSuchAlgorithmException {
+        this(data, false);
     }
 
-    public static MockedTorrent fromFile(final File torrent) throws IOException, NoSuchAlgorithmException {
-        final byte[] data = FileUtils.readFileToByteArray(torrent);
-        return new MockedTorrent(data, true);
```

Comment:
Any idea why this used to default to `true`? Can't see why we'd want to have the `seeder` value as true, as that marks our torrent as the first/initial seeder in the swarm.

Reply:
No reason idead. You can also remove the second parameter from the `MockedTorrent` constructor as well.